bbschema/entity.py

```diff
@@ -343,10 +343,11 @@ def update(self, identifier_json):
 
         if 'value' in identifier_json:
             new.value = identifier_json['value']
-        if 'identifier_type_id' in identifier_json:
+
+        if 'identifier_type' in identifier_json:
             new.identifier_type_id =\
-                identifier_json.get('identifier_type', {}).\
-                get('identifier_type_id')
+                identifier_json.get('identifier_type').\
```

Comment:
I don't see a point in using the second argument of get.
It s already checked here in the previous line
```python
        if 'identifier_type' in identifier_json:
```